koku/api/report/queries.py

```diff
@@ -716,7 +737,7 @@ def annotations(self):
             (Dict): query annotations dictionary
 
         """
-        pass
+        raise NotImplementedError('Annotations must be defined by sub-classes.')
```

Comment:
This improves behavior and testability. It also helps document the intended use.